tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,14 +87,27 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        var stdout = wsk.cli(Seq("property", "get", "--auth")).stdout
+        stdout should include regex ("""(?i)whisk auth\s*$""")      // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--apihost")).stdout
+        stdout should include regex ("""(?i)whisk API host\s*$""")  // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--namespace")).stdout
+        stdout should include regex ("""(?i)whisk namespace\s*_$""")// default = _
+    }
+
     it should "show api build version" in {
+        wsk.cli(wskprops.overrides ++ Seq("property", "set"))
         val stdout = wsk.cli(wskprops.overrides ++ Seq("property", "get", "--apibuild")).stdout
         stdout should include regex ("""(?i)whisk API build\s+201.*""")
+        wsk.cli(Seq("property", "unset", "--auth", "--apihost", "--apiversion", "--namespace"))
```

Comment:
I'll add a couple helper methods in the TestUtils class to backup and restore the .wskprops file.  We can use these instead of the WSK_CONFIG_FILE env.  We may need to fix other Java tests that are more intrusive..